.projenrc.js

```diff
@@ -0,0 +1,101 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
+const { AwsCdkConstructLibrary, ProjectType, NodePackageManager } = require("projen");
+
+const project = new AwsCdkConstructLibrary({
+  name: "datadog-cdk-constructs",
+  description: "CDK Construct library to automatically instrument python and node functions with datadog tracing",
+  author: "Datadog",
+  authorOrganization: true,
+  entrypoint: "lib/index.js",
+  repositoryUrl: "git@github.com:DataDog/datadog-cdk-constructs.git",
+
+  projectType: ProjectType.LIB,
+  packageManager: NodePackageManager.YARN,
+
+  jsiiFqn: "projen.AwsCdkConstructLibrary",
+  defaultReleaseBranch: "main",
+  releaseEveryCommit: false,
+  cdkDependenciesAsDeps: false,
+  cdkVersion: "1.71.0",
```

Comment:
How did you arrive at this version number? Was this here before?

Reply:
This defines to minimum version number for all the CDK dependencies in the project, I came to this number because in the DD-CDK without Projen all the CDK dependencies were "^1.71.0". Basically I was trying to match the old (before projen) package.json. https://github.com/DataDog/datadog-cdk-constructs/blob/63329136738f353d0e862d0d402f2ad1298d1862/package.json#L19